manifests/s/Soundly/Soundly/2021.11.0.0/Soundly.Soundly.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Soundly.Soundly
+PackageVersion: 2021.11.0.0
+PackageLocale: en-US
+Publisher: Soundly

```

```suggestion
Publisher: '"Soundly"'
```
'"Soundly"' in YAML will output `"Soundly"` instead of `Soundly`.

This was done to match Control Panel.